src/pages/methods/workflow-details/WorkflowWrapper.ts

```diff
@@ -289,6 +293,24 @@ export const WorkflowsContainer = (props: WorkflowContainerProps) => {
         setPermissionsModalOpen,
         refresh: loadSnapshot,
       }),
+    showCloneModal &&
```

Comment:
One thing to maybe discuss during the UX review today, is that the button shows `clone snapshot`, but then it opens a `clone method` modal 🤔 
It does not seem like there is a difference between a snapshot or a method, is there?

Reply:
That is a good point. It is actually cloning the snapshot (of that method) that is selected from the dropdown. I think it might make more sense to call everything `Clone snapshot` instead. I am going to change that and based on the outcome of UX discussion it can be changed again later.